lib/simplifyExpression/collectAndCombineSearch/LikeTermCollector.js

```diff
@@ -220,6 +223,19 @@ function getTermsForCollectingMultiplication(node) {
 }
 
 // A helper function for getTermsForCollectingMultiplication
+// e.g. nthRoot(x, 2), append '2': nthRootNode to terms dictionary
```

Comment:
I would make a constant value string like CONSTANT at the top of the file, but for nthroot

and then make the key `NTHROOT +  rootNodeValue` or something - makes it easier to debug cause you know it's referring to roots without having to have seen this code